static/app/components/modals/widgetViewerModal.tsx

```diff
@@ -47,8 +49,12 @@ const HALF_TABLE_ITEM_LIMIT = 10;
 const GEO_COUNTRY_CODE = 'geo.country_code';
 
 function WidgetViewerModal(props: Props) {
-  const {organization, widget, selection, location, Footer, Body, Header, onEdit} = props;
+  const {Footer, Body, Header, widget, onEdit, organization, selection, location} = props;
   const isTableWidget = widget.displayType === DisplayType.TABLE;
+  const [modalSelection, setModalSelection] = React.useState<PageFilters>(
+    cloneDeep(selection)
```

Comment:
Do you need a deep clone?

Reply:
Hey Evan, probably not. We're not mutating the pageFilter anywhere so it should be safe to use selection directly. I've updated to reflect this!